devel/n/Portfile

```diff
@@ -3,19 +3,19 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tj n 9.0.0 v
+github.setup        tj n 9.0.1 v
 categories          devel
 platforms           darwin
-maintainers         nomaintainer
+maintainers         {@akierig fastmail.de:akierig}
```

Comment:
can "openmaintainer" be added as well to indicate that others can make minor changes to the Portfile without your explicit approval. Ideally, this should be added unless there are compelling reasons not to.

Reply:
done!